packages/app-project/.storybook/specStory.js

```diff
@@ -0,0 +1,8 @@
+import { composeStory } from '@storybook/react'
+import preview from './preview.js'
+
+export const SpecStory = (StoryComponent) => {
+  return composeStory(StoryComponent, preview)
```

Comment:
This needs to support passing an optional `Meta` object to `composeStory`. `Meta` contains any parameters and decorators that an individual component's stories use.

Reply:
I think I would use the spread operator to decorate a function with additional args. 

```js
function composeStoryWithAnnotations(...args) {
  return composeStory(...args, annotations)
}
```

Reply:
Good call - pushed the change